src/cmd_line/commands/write.ts

```diff
@@ -37,7 +37,7 @@ export class WriteCommand extends node.CommandBase {
     return this._arguments;
   }
 
-  execute(modeHandler : ModeHandler) : void {
+  async execute(modeHandler : ModeHandler) : Promise<void> {
```

Comment:
Actually, this one doesn't need to be async because you don't do any awaiting inside it.